touchforms/formplayer/api.py

```diff
@@ -295,21 +296,22 @@ def post_data_helper(d, auth, content_type, url, log=False):
 
 
 def formplayer_post_data_helper(d, auth, content_type, url):
-    data = json.dumps(d)
+    data = json.dumps(d).encode('utf-8')
     up = urlparse(url)
-    logging.info("Request to url: %s" % up.geturl())
     headers = {}
     headers["Content-Type"] = content_type
     headers["content-length"] = len(data)
     headers["Cookie"] = 'sessionid=%s' % settings.FORMPLAYER_INTERNAL_AUTH_KEY
```

Comment:
Could comment here that this is to be removed once formplayer supports the mac digest.

Reply:
updated https://github.com/dimagi/touchforms/pull/306/commits/e547a71491918437cdd2c4799203fa9232b62696